docs/Getting Started/Example Apps/index.md

```diff
@@ -123,3 +127,39 @@ The [Hello Sdl JavaEE](https://github.com/smartdevicelink/sdl_java_suite/tree/ma
 To connect the sample app to the infotainment system, please follow the instructions in the [Connecting to an Infotainment System](Getting Started/Connecting to an Infotainment System) guide.
 !@
 
+@![javascript]
+## Connecting to an Infotainment System
+### Connecting as a WebSocket Client
+For plain JavaScript SDL apps, connecting as a WebSocket client requires a version of SDL Core that can accept incoming WebSocket connections (at least 6.1.0). If connecting using Manticore, ensure that the version of SDL Core that Manticore supports is at least 6.1.0. A workaround to this limitation is to use a proxy program that your app connects to which modifies the incoming WebSocket connection into a TCP connection, which then connects to SDL Core on the app's behalf. A `proxy.jar` Java program that does exactly this is available in the [repository's JavaScript example folder](https://github.com/smartdevicelink/sdl_javascript_suite/blob/master/examples/js/hello-sdl). Check the `readme.md` for how to run it. This workaround for the lower versions of Core is also necessary for WebEngine apps. Please check the [Connecting to an Infotainment System](Getting Started/Connecting to an Infotainment System) guide for more detailed instructions on how to get the emulator's IP address and port number.
```

Comment:
```suggestion
For vanilla JavaScript SDL apps, connecting as a WebSocket client requires a version of SDL Core that can accept incoming WebSocket connections (at least v6.1.0). If connecting using Manticore, ensure that the version of SDL Core that Manticore supports is at least v6.1.0. A workaround to this limitation is to use a proxy program that your app connects to which modifies the incoming WebSocket connection into a TCP connection, which then connects to SDL Core on the app's behalf. A `proxy.jar` Java program that does exactly this is available in the [repository's JavaScript example folder](https://github.com/smartdevicelink/sdl_javascript_suite/blob/master/examples/js/hello-sdl). Check the `readme.md` for how to run it. This workaround for older versions of Core is also necessary for WebEngine apps. Please check the [Connecting to an Infotainment System](Getting Started/Connecting to an Infotainment System) guide for more detailed instructions on how to get the emulator's IP address and port number.
```

Don't tell devs to check if manticore supports it. Either manticore does or doesn't. If it doesn't right now, then remove the sentence and add an issue to add it when manticore is updated.

Reply:
https://github.com/smartdevicelink/sdl_app_library_guides/issues/274